rustls/src/crypto/hpke.rs

```diff
@@ -26,6 +26,7 @@ pub trait HpkeProvider: Debug + Send + Sync + 'static {
 }
 
 /// An HPKE suite, specifying a key encapsulation mechanism and a symmetric cipher suite.
+#[derive(Debug, Clone)]
```

Comment:
Tiny nit: maybe alphabetize these if you're newly adding them?

Reply:
Alphabetized the new derives throughout.